src/containers/PageBuilder/Primitives/SearchCTA/FilterCategories/FilterCategories.module.css

```diff
@@ -0,0 +1,146 @@
+@import '../../../../../styles/customMediaQueries.css';
+
+.root {
+  width: 100%;
+  height: 100%;
+}
+
+.dropdownContainer {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  display: grid;
+  user-select: none;
+  margin-top: 4px;
+  padding: 0 12px;
+  @media (--viewportMedium) {
+    padding: 0 16px;
+    margin: 0;
+  }
+  @media (--viewportLarge) {
+    padding: 0;
+  }
+}
+
+.toggleButton {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 40px;
+  border-radius: 4px;
+  cursor: pointer;
+  color: var(--colorGrey700);
+  transition: all 0.1s;
+  font-weight: 400;
+
+  @media (--viewportLarge) {
+    margin: 0;
+    padding: 10px 20px 10px 16px;
+  }
+
+  @media (--viewportMedium) {
+    height: 48px;
+  }
+}
+
+.dropdownContent {
+  position: absolute;
+  top: 51px;
+  left: 50%;
+  transform: translateX(-50%);
+  width: calc(100% + 16px);
+  min-width: 288px;
+  padding: 14px 0 18px;
+  background-color: white;
+  border-radius: 4px;
+  box-shadow: var(--boxShadowPopup);
+  z-index: 1;
+
+  @media (--viewportLarge) {
+    left: -8px;
+    transform: none;
+    width: 100%;
+  }
+}
+
+.option {
+  position: relative;
+  width: 100%;
+  padding: 7px 0 7px 22px;
```

Comment:
There needs to be padding also on the right-hand side.
![Screenshot 2025-04-23 at 21 58 07](https://github.com/user-attachments/assets/71af7715-46d1-4f88-bf7a-747281ed1e71)